docs/tutorials/getting-started/tut-setup-dev.md

```diff
@@ -382,19 +383,80 @@ In order to create a branch, use the  `git checkout -b [branch_name]` command, w
 Switched to a new branch 'my_integration_name'
 ```
 
-Now, create a directory under `Integrations`, named after your integration where you will put all your integration files later, and add it to the staged changes in `git`. 
+Now, create a directory under `￿Packs/<Your pack name>`, named after your product where you will put all your content files later, and add it to the staged changes in `git`.  
+For more description regarding what exactly a pack is please click [here](packs-fromat). 
 
-Make sure you use **PascalCase** in the directory name (i.e. `MyIntegration`):
+Make sure you use **PascalCase** in the directory name (i.e. `MyIntegration`), you can create a Pack and an Integration directory using the `demisto-sdk init` command.
 
+An example of creating a pack called test, with an integration called test, and with the metadata file created automatically: 
 ```bash
-(venv) sb@dddd:~/demisto/content$ mkdir Integrations/MyIntegration
-(venv) sb@dddd:~/demisto/content$ git add Integrations/MyIntegration
+➜  content-docs2 git:(add-pack-and-sdk-docs) ✗ demisto-sdk init --pack 
+Please input the name of the initialized pack: test
```

Comment:
not good example for partner tutorial.
it should be as if partner run this command.
`Support type of the pack:` is not option `1` but `2`
insert meaningful names , you can use existing partner integration or not.
But insert contact details, url to support, etc.
`Server min version: 5.0.0`